### ISSUE TITLE
Duda enunciado 7.4 parametros.py

### ISSUE THREAD

**Issue by elisaorchard** [IIC2233/Syllabus#351]
* [ X] Leí las reglas del foro (https://github.com/IIC2233/Syllabus/wiki/3.-Reglas-del-foro-(issues)).
* [X ] Revisé el compilado de dudas de la tarea y no encontré una *issue* similar a la mía ([click aquí para ir al compilado](/IIC2233/Syllabus/issues/11)).
* [ X] Mi duda no se trata sobre una librería o *built-in*, ya que eso se pregunta en la *issue* creada para ello ([click aquí para ir a esa _issue_](/IIC2233/Syllabus/issues/10)).
* [X ] De ser necesario, colocaré código simple que permita explicar mi problema o duda, ya que compartir código de esta evaluación puede ser considerado una falta a la ética.

### Duda

Hola. Estaba leyendo la parte de archivos del enunciado, y hay algo que todavía no entiendo. Con respecto al archivo parametros.py, el enunciado dice que "es fundamental incluir cualquier valor constante necesario en tu tarea, así como cualquier tipo de path utilizado". 

A qué se refiere con cualquier tipo de path utilizado? Esto es incluso si ocupé paths en otro módulo?

**Comment by augustayanez** [comment]
Exacto! Cualquier path que quieras usar en cualquiera de tus módulos, debes guardarlo como una variable en parametros.py e importarlo al momento de abrir el archivo.
